pkg/services/rpcsrv/server.go

```diff
@@ -1531,9 +1531,10 @@ func (s *Server) verifyProof(ps params.Params) (any, *neorpc.Error) {
 	}
 	vp := new(result.VerifyProof)
 	val, ok := mpt.VerifyProof(root, p.Key, p.Proof)
-	if ok {
-		vp.Value = val
+	if !ok {
+		return nil, neorpc.ErrInvalidProof
```

Comment:
`docs/rpc.md` should be updated to reflect this.

Reply:
Yeah, combining 3cab342b40e904f2a6c29fc7d712ebeeba78f57b and 60167c39b377d3c188f894885abb8926b5fa3a6d makes sense, it's the same logical change.